src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizQuestion.java

```diff
@@ -68,6 +68,13 @@ public abstract class QuizQuestion extends DomainObject {
     @JsonIgnore
     private QuizExercise exercise;
 
+    @ManyToOne
+    private QuizGroup quizGroup;
```

Comment:
The ManyToOne relation is removed and changed to QuizGroupId instead since we still need to map a QuizQuestion to a QuizGroup while saving the QuizPool.